src/routes/api/user/user.route.js

```diff
@@ -58,8 +58,9 @@ router.put('/update/:email', [validateToken, confirmEmaiAuth], UserController.up
 router.post('/signup/admin', [validateToken, admin, confirmEmaiAuth], UserController.createAdmin);
 router.post('/signout', validateToken, UserController.signoutUser);
 router.post('/profiles/:userId/follow', [validateToken, validateUserId], followController.follow);
-router.get('/profiles/following', validateToken, followController.listOfFollowedUsers);
-router.get('/profiles/followers', validateToken, followController.listOfFollowers);
+// router.get('/profiles/following', validateToken, followController.listOfFollowedUsers);
+// router.get('/profiles/followers', validateToken, followController.listOfFollowers);
+router.get('/profiles/:followersOrFollowing', validateToken, followController.listOfFollowersOrFollowed);
```

Comment:
The previous way was better in the case of routing because the 2 routes are different and therefore should have different methods.

Reply:
In terms of conventions too. e.g look at twitter